contracts/foundry.toml

```diff
@@ -0,0 +1,7 @@
+
+[profile.default]
+src = 'src'
+out = 'out'
+libs = ['../node_modules', 'lib']
+
+# See more config options https://book.getfoundry.sh/reference/config.html
```

Comment:
The link is broken. The correct one should be https://book.getfoundry.sh/reference/config/